pkg/db/table.go

```diff
@@ -274,7 +274,7 @@ func setImplicitColumn(table *model.TableInfo) {
 		ID:   ImplicitColID,
 		Name: model.NewCIStr(ImplicitColName),
 	}
-	newColumn.Tp = mysql.TypeInt24
+	newColumn.Tp = mysql.TypeLonglong
```

Comment:
why not using `model.NewExtraHandleColInfo()`?

Reply:
this need update vendor, I will do next pr